app/javascript/packs/elm.js

```diff
@@ -2,6 +2,7 @@ import { Elm } from "../elm/src/Main.elm";
 import logoNav from "../../assets/images/aircasting-logo-nav.svg";
 import linkIcon from "../../assets/images/link-icon.svg";
 import resetIcon from "../../assets/images/reset-icon.svg";
```

Comment:
Is there no way to load the image in elm directly? It would probably simplify a lot. And we should figure it out at some point if we will move more logic to elm and need to load there more images.

Reply:
I guess this was answered offline